src/document/url.ml

```diff
@@ -253,7 +253,7 @@ module Anchor = struct
     | None -> assert false (* We got a root, should never happen *)
     | Some page ->
         let anchor = Printf.sprintf "%s-%s" (Path.string_of_kind kind) name in
-        { page; anchor; kind = (kind :> kind) }
+        { page; anchor; kind :> kind }
```

Comment:
Didn't know this was allowed!

Reply:
Not sure about the compatibility. I'll check.

Reply:
This is a syntax error on 4.02. This is a bug!

Reply:
Fixed in https://github.com/ocaml-ppx/ocamlformat/pull/2388